### DISCUSSION TITLE
Add overlay on a background images

### DISCUSSION THREAD

**Discussion by StefanoTesla** [tailwindlabs/tailwindcss#6857] [Help]
Hello everyone, I have one question, I'm converting a site and I need to add a gradient from white to transparent on a background image.

Looking in the documentation I've seen bg-gradient, but this will override my background image...any solution?? custom css??

**Comment by alexanderbluhm** [comment]
Hi @StefanoTesla 
It is possible doing that with background image, but the easier solution is to use an `img` element and add a gradient on top like this: https://play.tailwindcss.com/jBBf3db6Sa?size=552x836

If you want the background solution you can look at [this](https://stackoverflow.com/questions/16589519/use-css-gradient-over-background-image)

**Comment by StefanoTesla** [comment]
mmmh...can be good, but I lost the responsive feature of the image background....I need to study with this...
